### DISCUSSION TITLE
How safely run magic mirror (node v10.x) and room assistant (node v 10.x) one one machine

### DISCUSSION THREAD

**Discussion by alexbell62** [mKeRix/room-assistant#372] [Q&A]
Hi 
I've got a rpi4 running magic mirror and was wondering if it is going to be safe to install room assistant on the same machine? Ie when running `node -v` it shows `v10.23.0` while my understanding for room assistant v12 is required. I wonder if there is an easy and safe way to run a magic mirror on `v10.x` and room assistant on `v12.x` simultaneously?

Thanks,
Alex

**Comment by mKeRix** [comment]
room-assistant actually also works and is tested on Node v10, so you should be able to run them side by side using the Node version you already have! But if you want to manage multiple Node versions I think [nvm](https://github.com/nvm-sh/nvm) would be the right tool for the job.

**Reply by alexbell62** [reply]
hi @mKeRix thanks for your advice! I just tried it today and it all seems to work fine!
